packages/sra-report/src/postman_environment_factory.ts

```diff
@@ -0,0 +1,74 @@
+import { SignedOrder, ZeroEx } from '0x.js';
+import { Schema, schemas as schemasByName } from '@0xproject/json-schemas';
+import * as _ from 'lodash';
+
+import * as kovanTokensEnvironmentJSON from '../postman_configs/environments/kovan_tokens.postman_environment.json';
+import * as mainnetTokensEnvironmentJSON from '../postman_configs/environments/mainnet_tokens.postman_environment.json';
+
+interface EnvironmentValue {
+    key: string;
+    value: string;
+    enabled: boolean;
+    type: string;
+}
+
+export const postmanEnvironmentFactory = {
```

Comment:
Could you add a comment here explaining what it does. It's not immediately clear to me.